ra-optimisation/rao-api/src/main/java/com/farao_community/farao/rao_api/RaoParameters.java

```diff
@@ -32,6 +34,7 @@
  * @author Baptiste Seguinot {@literal <baptiste.seguinot at rte-france.com>}
  */
 public class RaoParameters extends AbstractExtendable<RaoParameters> {
+    static final Logger LOGGER = LoggerFactory.getLogger(RaoParameters.class);
```

Comment:
not sure it is used (it is you only change in this file)

Reply:
true